Automattic-Tracks-iOS/TracksService.m

```diff
@@ -54,7 +54,7 @@ @interface TracksService ()
 NSString *const TracksUsernameKey = @"_ul";
 
 NSString *const TracksUserTypeAnonymous = @"anon";
-NSString *const TracksUserTypeWPCOM = @"wpcom:user_id";
+NSString *const TracksUserTypeAuthenticated = @"wpcom:user_id";
```

Comment:
The Simplenote app will set the `authenticatedUserTypeKey` parameter with something pre-determined, much like how `eventNamePrefix` works to override `wpios`.